rosa_getting_started/rosa-getting-started.adoc

```diff
@@ -0,0 +1,48 @@
+[id="rosa-getting-started"]
+= Getting started with {product-title}
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-getting-started
+
+toc::[]
+
+Follow this getting started document to quickly create a {product-title} (ROSA) cluster, add users, deploy your first application, and learn how to scale and delete your cluster.
```

Comment:
Unordered list?

Reply:
Thanks for this suggestion Neal. For the reasons outlined in the earlier thread, I am going to leave this one as is.